README.md

```diff
@@ -14,10 +14,13 @@ The project operates on an open contribution model, welcoming contributions from
 To see what's incoming, check the open [pull requests](https://github.com/greenelab/deep-review/pulls).
 For project discussion and planning see the [Issues](https://github.com/greenelab/deep-review/issues).
 
-### Current stage: resubmitted to _JRSI_ and _bioRxiv_
+### Current stage: planning Deep Review 2
```

Comment:
Do you want to tag it with a year like _2019_ to make it clearer when you plan for the next release? I am also fine with 2, so this is just a suggestion.